src/player.js

```diff
@@ -276,6 +278,28 @@ class Player extends FakeEventTarget {
   }
 
   /**
+   * @function _onPlay
+   * @return {void}
+   * @private
+   */
+  _onPlay(): void {
+    this._eventManager.unlisten(this, Html5Events.PLAY);
```

Comment:
This will cause the player to stop listening to play event altogether.
Better have a handler with a flag, so we can also query this from outside, e.g.  if playback has ever started before.